common/djangoapps/student/tests/test_verification_status.py

```diff
@@ -371,7 +371,7 @@ def _assert_course_verification_status(self, status):
         # Verify that the correct banner color is rendered
         self.assertContains(
             response,
-            "<article class=\"course {}\">".format(self.MODE_CLASSES[status])
+            "<article class=\"course {}\"".format(self.MODE_CLASSES[status])
```

Comment:
Is this intended?

Reply:
Yes, now we have an id in that selector so the check is just for the first part of the div

Reply:
👍